test/e2e/volume_provisioning.go

```diff
@@ -350,14 +358,27 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 	})
 })
 
-func verifyDefaultStorageClass(c clientset.Interface, expectedDefault bool) {
-	sc, err := c.StorageV1().StorageClasses().Get("default", metav1.GetOptions{})
+func getDefaultStorageClassName(c clientset.Interface) (name string, found bool, err error) {
```

Comment:
Given how this function is used, I'm not sure why it doesn't fail on error or not found rather than putting that responsibility on every call site.